AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -5376,6 +5376,8 @@ tv5unis.ca#%#//scriptlet('set-cookie', 'tv5-hide-cookies-notification', 'true')
 pushsquare.com#%#//scriptlet("set-constant", "dgpConsentType", "undefined")
 hotcleaner.com#%#(function(){if(-1==document.cookie.indexOf("CONSENT")){var c=new Date, d=c.getTime(), date = new Date(d + 365 * 60 * 60 * 1000); document.cookie="CONSENT=301212NN; path=/; expires=" + date.toUTCString(), location.reload()}})();
 berndes.com#%#//scriptlet('set-cookie', 'cookie-preference', '1')
+conrad.at#%#//scriptlet("set-cookie", "cookiesGloballyAccepted", "yes")
+conrad.at#%#//scriptlet("set-cookie", "cookieNotificationClosed", "yes")
```

Comment:
We have an old rule that can be slightly changed.
`||conrad.*/etc.clientlibs/ccp-sites/components/structure/cookie-notification/clientlibs.*.js`